.github/workflows/perf.yml

```diff
@@ -9,6 +9,7 @@ on:
 
 env:
   terraform_version: '0.15.3'
+  openresty_version: '1.19.3.1'
```

Comment:
Is there a way for us to pick it from `.requirements` instead?

Reply:
added and verified!